x/merkledb/trieview.go

```diff
@@ -391,13 +391,13 @@ func (t *trieView) getProof(ctx context.Context, key []byte) (*Proof, error) {
 // [maxLength] must be > 0.
 func (t *trieView) GetRangeProof(
 	ctx context.Context,
-	start, end []byte,
+	start, end Maybe[[]byte],
```

Comment:
nit: I noticed that in some places we add the type to both `start` and `end`. In other places, we do `start, end Maybe[[]byte]`...would be nice to unify this for style consistency.